src/state/swap/hooks.ts

```diff
@@ -235,27 +239,18 @@ export function useDerivedSwapInfo(platformOverride?: RoutablePlatform): UseDeri
     }
 
     // Use a static version
-    const staticProvider = provider ? new StaticJsonRpcProvider(provider?.connection.url) : undefined
+    const staticProvider = provider ? new StaticJsonRpcProvider(provider.connection, provider.network) : undefined
 
-    const getTrades = isExactIn
-      ? getExactInFromEcoRouter(
-          {
-            currencyAmountIn: parsedAmount,
-            currencyOut: outputCurrency,
-            ...commonParams,
-          },
-          ecoRouterSourceOptionsParams,
-          staticProvider
-        )
-      : getExactOutFromEcoRouter(
-          {
-            currencyAmountOut: parsedAmount,
-            currencyIn: inputCurrency,
-            ...commonParams,
-          },
-          ecoRouterSourceOptionsParams,
-          staticProvider
-        )
+    console.log('useDerivedSwapInfo: fetching trades')
```

Comment:
do we need this console log?

Reply:
Don't think so, but they wouldn't hurt anyways.